lib/swoosh/adapters/mailgun.ex

```diff
@@ -134,5 +134,5 @@ defmodule Swoosh.Adapters.Mailgun do
      |> Kernel.++(inline)
     }
   end
-  defp encode_body(no_attachments), do: Plug.Conn.Query.encode(no_attachments)
+  defp encode_body(no_attachments), do: URI.encode_query(no_attachments)
```

Comment:
Why did this change?

Reply:
@princemaple was suggesting to change that also in the mailgun adapter (see [this](https://github.com/swoosh/swoosh/pull/188#discussion_r156606823) comment  )